services/app-web/src/common-code/dateHelpers/calendarDate.ts

```diff
@@ -1,12 +1,21 @@
 import { dayjs } from './dayjs'
 
-// We store calendar dates (dates with no time info) in UTC for consistency.
-// This formats a date correctly like '12/02/2022' for display.
-function formatCalendarDate(date: Date | undefined): string {
-    if (!date) {
+// We store calendar dates in UTC for consistency. This formats a date like '12/02/2022'.
+// These formatted dates, should only be used for display, never saved to the database.
+function formatCalendarDate(date: Date | undefined | string, timeZone?: 'UTC' | 'America/New_York'): string {
+    const tz = timeZone || 'UTC'
+    if (!date || !dayjs(date).isValid()) {
+        return ''
+    }
+    return dayjs(date).tz(tz).format('MM/DD/YYYY')
+}
+
+function formatDateTime(date: Date | undefined | string, timeZone?: 'UTC' | 'America/New_York'): string {
```

Comment:
I think date times displayed are always Eastern in the application going forward. |

That means this can be `formatEasternDateTime` and remove timezone handling all together. Also, I suggest adding the appended `ET` string here as well, that's part of the date time display.